### PR TITLE
feat: add z3str3 soundness issue

### PR DESCRIPTION
**Pull request by nicdard**

### CODE REVIEW

File: bugs/README.md [hunk 1]
```diff
@@ -7,4 +7,5 @@
 | [QF_NRA 1](z3/qf_nra-1.smt2) | QF_NRA | Segfault            | [#6044](https://github.com/Z3Prover/z3/issues/6044)                         | 20/05/22 |
 | [QF_S 1](z3/qf_s-1.smt2)     | QF_S   | Memory Leak         | [#6076](https://github.com/Z3Prover/z3/issues/6076)                         | 04/06/22 |
 | [QF_BV 1](z3/qf_bv-1.smt2)   | QF_BV  | Assertion Violation | [#6143](https://github.com/Z3Prover/z3/issues/6143)                         | 07/07/22 |
+| [QF_S_2](z3/qf_s-2.smt2)     | QF_S   | Soundness           | [#6159](https://github.com/Z3Prover/z3/issues/6159)                         | 14/07/22 |
```
Comment (lweitzendorf) [inline bugs/README.md:10]: For consistency, remove the last underscore in the first column

**Review by lweitzendorf — CHANGES_REQUESTED** [review]

File: bugs/README.md (hunk 1 above)
Comment (lweitzendorf) [inline bugs/README.md:10]:
```suggestion
| [QF_S 2](z3/qf_s-2.smt2)     | QF_S   | Soundness           | [#6159](https://github.com/Z3Prover/z3/issues/6159)                         | 14/07/22 |
```

**Review by lweitzendorf — CHANGES_REQUESTED** [review]

File: bugs/README.md (hunk 1 above)
Comment (nicdard) [reply bugs/README.md:10]: Sure I didn't notice!

Comment (nicdard) [reply bugs/README.md]: Should be ok now

**Pull request merged**